### ISSUE TITLE
check node version at start of flynt create

### ISSUE THREAD

**Issue by emcarru** [flyntwp/flynt-cli#82]
The node version is only checked half way through create, meaning if you have the wrong version, you have to manually run the remaining steps manually.

**Comment by bdbch** [comment]
Yeah this is annoying haha. I almost freaked out when I wanted to create a new project and because of the Windows magic I had some issues which I had to resolve first (mysql host, node version, etc.) and everytime the create process failed, I had to restart again and re-enter all of my data in the cli prompt.

Maybe there could be a way to save the current state of the installation when the create/setup step fails so you can just continue the setup via some command (for example `flynt continue`.
